# This is an action definition file, which has three parts: the goal
# the result, and the feedback.
# Part 1: the goal.
#
# The target path to change EXIF Data
string path
int64 timestamp_in_milliseconds
---
# Part 2: the result, sent by action server unpon completion
#
# Image successfully changed or not
bool success
---
# Part 3: the feedback,to be sent periodically by server
#
# The current operation
string operation
